### ISSUE TITLE
smb1 query re samba and "building a Pi based nas"

### ISSUE THREAD

**Issue by hydra3333** [thagrol/Guides#11]
Hello.
In 2.4 it mentions bullseye has smb v1 disabled.

Does that mean I don't need to install smb v1 on Win10 clients nor install wssd  on the Pi ?

**Comment by thagrol** [comment]
You only need to enable smb v1 on client or server if you want to use it.

Without smbv1 or wsdd installed on the Pi, network browsing from windos will not show your Pi or its shares. The secure option is to leave smbv1 disabled and to install wsdd on the Pi.

**Comment by hydra3333** [comment]
OK.  An update for you which hopefully makes things easier: 

With a freshly installed x64 Pi OS, I installed samba and just before fiddling with wsdd I tried to connect from a Win10 PC (i.e. no SMB v1 on either the PC nor the Pi).

Using only \\10.0.1.6\ as the address, windows explorer showed the shares and then I could browse and folder trees on the shares to the NTFS disks and see the files and play the media files in a media player on the PC.

Hope that helps :)
